dev-guide/deploy.md

```diff
@@ -0,0 +1,13 @@
+# Build for deployment
+
+The easiest way to deploy TiDB is to use Docker Compose, see [TiDB Docker Compose Deployment](../dev/how-to/get-started/deploy-tidb-from-docker-compose.md). For more deployment methods, see [TiDB deployment methods](../_index.md).
+
+## Before you begin
```

Comment:
It looks a little weird to me that here says "Before you begin" and the first sentence in the section says "Before you start". It's duplicated and inconsistent.